cnf/ext/repositories.bnd

```diff
@@ -7,6 +7,11 @@ mavencentral:           https://repo.maven.apache.org/maven2
         releaseUrl="${mavencentral}";\
         index="${.}/central.mvn";\
         readOnly=true,\
+    aQute.bnd.repository.maven.provider.MavenBndRepository;\
+        name="Sonatype";\
+        snapshotUrl="https://oss.sonatype.org/content/repositories/snapshots/";\
+        index="${.}/sonatype.mvn";\
+        readOnly=true,\
```

Comment:
Can we not fold this into "OSGi Snapshots"? After all the osgi-test artifacts are from OSGi.

Reply:
I kept it separate for resolving - the OSGi Snapshot repo is not included in -runresolve, but Sonatype is. I suppose i could use -runblacklist instead to exclude just those we want to exclude.

Reply:
OK, that makes sense. But then "Sonatype" is not a descriptive name for the new repo.

Reply:
It is about as descriptive as "Maven Central"...

Im hoping that the release of osgi-test is not far away and i can add them to Maven Central. Actually, i think i settled on an implementation which didn't use osgi-test so maybe i can ditch it altogether.

Reply:
My point was that you segregated those artifacts into the separate repo so you can use that repo in resolving. I could not see that, so I asked why. Perhaps a name like "Test Resolve" or something would better convey the intent of the repo and stop someone else in the future adding something to the repo which we don't want in the test resolve.

Reply:
Sorry, I didn't explain it very well. My point was, as we have a generic "Maven Central" repo where we throw all of the bundles that live on Maven Central (and don't belong in another repo), I thought it might be nice to have a generic "Sonatype" repo where we throw all of the snapshots that don't live anywhere else. The purpose of segregating was more about keeping the OSGi Snapshot repo *out* of the resolve (because it seems to exclusively contain build-time-only bundles) than about having a specific repo for resolving the integration test.

Reply:
I have left this one as it is for now. Longer term (once osgi-test goes into a proper release) we can either move osgi-test into Maven Central, or split Maven Central into "Maven Central" and "Maven Central Testing" for test-oriented dependencies and make osgi-test a member of that.

Reply:
I found that the dependency that I was looking for in osgi-test has an equivalent in `aQute.lib.exceptions`, so the latest push has removed osgi-test (and the Sonatype repo) completely.